test/cypress/tests/modules/BlockEvents.cy.ts

```diff
@@ -1,5 +1,25 @@
+import type EditorJS from '../../../../types/index';
```

Comment:
Maybe move some helper functions and mocks to a separate file and apply refactoring where possible (extract common fragments). Too hard too read atm

Reply:
I decided to leave small mock classes inside tests for better understanding of test implementation. If I'll move them to externall files, they won't be such self-descriptive.

I can separate this file to several files to make them smaller